endpointmanager/pkg/endpointmanager/postgresql/healthitproductstore.go

```diff
@@ -181,6 +176,96 @@ func (s *Store) GetHealthITProductsUsingVendor(ctx context.Context, vendorID int
 	defer rows.Close()
 
 	for rows.Next() {
+		var hitp endpointmanager.HealthITProduct
+		var locationJSON []byte
+		var certificationCriteriaJSON []byte
+		var vendorIDNullable sql.NullInt64
+		var practiceTypeString sql.NullString
+
+		err = rows.Scan(
+			&hitp.ID,
+			&hitp.Name,
+			&hitp.Version,
+			&vendorIDNullable,
+			&locationJSON,
+			&hitp.AuthorizationStandard,
+			&hitp.APISyntax,
+			&hitp.APIURL,
+			&certificationCriteriaJSON,
+			&hitp.CertificationStatus,
+			&hitp.CertificationDate,
+			&hitp.CertificationEdition,
+			&hitp.LastModifiedInCHPL,
+			&hitp.CHPLID,
+			&practiceTypeString,
+			&hitp.CreatedAt,
+			&hitp.UpdatedAt)
+		if err != nil {
+			return nil, err
+		}
+
+		ints := getRegularInts([]sql.NullInt64{vendorIDNullable})
+		hitp.VendorID = ints[0]
+
+		if !practiceTypeString.Valid {
+			hitp.PracticeType = ""
+		} else {
+			hitp.PracticeType = practiceTypeString.String
+		}
+
+		err = json.Unmarshal(locationJSON, &hitp.Location)
+		if err != nil {
+			return nil, err
+		}
+
+		err = json.Unmarshal(certificationCriteriaJSON, &hitp.CertificationCriteria)
+		if err != nil {
+			return nil, err
+		}
+
+		hitps = append(hitps, &hitp)
+	}
+
+	return hitps, nil
+}
+
+// GetHealthITProductsUsingName returns a slice of active HealthITProducts that were created by the given name
+func (s *Store) GetActiveHealthITProductsUsingName(ctx context.Context, name string) ([]*endpointmanager.HealthITProduct, error) {
```

Comment:
We should add a test or two in the `healthitproductstore_test.go` file for this new function